src/lib/functions/array/is_empty.ts

```diff
@@ -0,0 +1,7 @@
+export function is_empty(value: string | any[] | undefined) {
+  return (
+    value === undefined ||
+    Array.isArray(value) && value.length === 0 ||
+    typeof value === "string" && value === null
+  )
+}
```

Comment:
I dont get it.

Why not use `array.length === 0` ?

Reply:
> I dont get it.
> 
> Why not use `array.length === 0` ?

you added this on django, but i think this is not bad at all
do you want to remove it?

Reply:
Hmm i dont get why its this complex. I think doing the length is better